spec/unit/blockscore/collection/member_spec.rb

```diff
@@ -1,7 +1,32 @@
 module BlockScore
   class Collection
```

Comment:
@tjchambers given that this style was used in all the specs, I don't think this should be a blocker towards merging this pull request. However I would recommend a follow-up PR that fixes the specs to use the suggested approach, and then using it in all specs going forward.